### ISSUE TITLE
Create Faq Scaffold

### ISSUE THREAD

**Issue by alexfinnarn** [CUCentralAdvancement/giving-backend#22]
There are about 47 FAQs on the current site. All of them look to have rather plain text for the body field vs. complicated HTML...but that might have been stripped in the View Data Export

**Acceptance Criteria**

- [x] `rails g scaffold Faq title slug:string:uniq category:integer`
- [x] Use enums for FAQ category https://naturaily.com/blog/ruby-on-rails-enum
- [x] Add FAQ seed data from current site
- [ ] Make sure the rich text area content transfers without losing any tags

**Comment by alexfinnarn** [comment]
Setting up the scaffold isn't hard but the rich text definitely doesn't translate to the Trix editor well. Options can be:

- Still use Trix editor for FAQ answer/body content knowing that someone will have to add styles later.
- Try and convert the current body content into something that can be imported to Trix as-is...this assumes you can do this without everything being an attachment.
- Add a separate temporary field to transfer body field content as-is and the API just spits it out without trying to convert anything. Editing the FAQ and adding content to the Trix editor field negates the temp field. 
- Use a different rich text editor than Trix and going against the Action Text convention.

**Comment by alexfinnarn** [comment]
> Try and convert the current body content in the Drupal 7 export into something that can be imported to Trix as-is...this assumes you can do this without everything being an attachment.

I did that route and with all the HTML preserved, it still can look a bit off and has links to `/node/12345` which will never be relevant in the new system. I think reviewing each FAQ individually is probably the best way to go. 

Also, I'm not sure the last time someone went through the FAQs. Several have been updated recently, but a majority of them have `09/15/2017` as the last updated date.
